red/Makefile

```diff
@@ -0,0 +1,11 @@
+STEPS = step0_rep.redl step1_read_print.red step2_eval.red step3_env.red step4_if_fn_do.red \
+		step5_tco.red step6_file.red step7_quote.red step8_macros.red step9_try.red stepA_mal.red
+
+all:
+	$(patsubst %.red,%,$(STEPS))
+

```

Comment:
Consider adding the `stats*` and `mal` targets as in other implementations. @kanaka do we have instructions for these?

Reply:
@dubek the stats mechanism now happens at the top-level using loccount so implementations no longer have `stats*` targets. The `mal` targets were really more of an experiment on my part to see how many implementations could be fully packaged into standalone executables. It can be really difficult depending on the target language. So I never documented it anywhere.